Examples/UIExplorer/UIExplorer.xcodeproj/project.pbxproj

```diff
@@ -572,6 +588,22 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
+		4CC373051B23ADA4004C3480 /* ShellScript */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/bash;
+			shellScript = "export ORIGIN=$(/usr/libexec/PlistBuddy -c \"Print :RCTWebSocketExecutor:origin\" \"${INFOPLIST_FILE}\")\nexport PORT=$(/usr/libexec/PlistBuddy -c \"Print :RCTWebSocketExecutor:port\" \"${INFOPLIST_FILE}\")\n\nif nc -w 5 -z localhost $PORT ; then\n  if ! curl -s \"${ORIGIN}:${PORT}/status\" | grep -q \"packager-status:running\" ; then\n    echo \"Port ${PORT} already in use, packager is either not running or not running correctly\"\n    exit 2\n  fi\nelse\n  # open w/t --args doesn't work:\n  # open $SRCROOT/../../packager/launchPackager.command --args 111 222 || echo \"Can't start packager automatically\"\n  osascript -e 'tell app \"Terminal\"\n    do script \"'$SRCROOT'/../../packager/launchPackager.command --origin='${ORIGIN}' --port='${PORT}'\"\n  end tell' || echo \"Can't start packager automatically\"\nfi";
```

Comment:
@frantic Sure, can do. What's a good place to place that shell script? This one?: `./scripts`.